File: src/easy/NumIdenticalPairs.swift
func numIdenticalPairs(_ nums: [Int]) -> Int {
  var count = 0
  var left = 0
  while left < nums.count {
    var right = left + 1
    while right < nums.count {
      if nums[left] == nums[right] {
        count += 1
      }
      right += 1
    }
    left += 1
  }
  return count
}

File: src/easy/num_identical_pairs.rs
pub fn num_identical_pairs(nums: Vec<i32>) -> i32 {
    let mut count: i32 = 0;
    let mut left: usize = 0;
    while left < nums.len() {
        let mut right: usize = left + 1;
        while right < nums.len() {
            if nums[left] == nums[right] {
                count += 1;
            }
            right += 1;
        }
        left += 1;
    }
    count 
}